site/docs/guide/basics.md

```diff
@@ -14,17 +14,17 @@ The easiest way to listen for messages is via
 
 ```ts
 bot.on("message", (ctx) => {
-  const message = ctx.message; // message object
+  const message = ctx.message; // the `Message` object
```

Comment:
I would prefer to use the intuitive understanding of context objects in comments and explanations, rather than referring to the technical term `Context` object (which btw is inaccurate because `Context` is a class, so we'd have to say instances of `Context` or objects of type `Context`).

Here, I'd stick with message objects rather than `Message` objects.

This makes English easier to read IMO.